sample-lnd.conf

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
nit: the commit messages seem to have a very long first line, which should be capped to around 80 characters.
See https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#model-git-commit-messages

I think you're probably just missing a double line break between the first line and the rest of the body.